src/go/cmd/token-vendor/app/tokenvendor.go

```diff
@@ -36,7 +36,7 @@ type PubKeyRepository interface {
 	// LookupKey retrieves the public key of a device from the repository.
 	// An empty string return indicates that no key exists for the given identifier or
 	// that the device is blocked.
```

Comment:
Can we document the two return values here?

Reply:
I only uploaded the PR to save it. What if I introduce a struct Key that has all the values?

Reply:
Done